droidlet/perception/semantic_parsing/nsp_transformer_model/train_model.py

```diff
@@ -537,6 +474,12 @@ def build_grammar(args):
     parser.add_argument(
         "--encoder_learning_rate", default=0.0, type=float, help="Learning rate for the encoder"
     )
+    parser.add_argument(
+        "--encoder_lr_schedules",
```

Comment:
can you make a more explicit comment about the algorithm here, and/or cite something if it comes from a paper?

Reply:
> can you make a more explicit comment about the algorithm here, and/or cite something if it comes from a paper?

Answered above.